docs/NSR.md

```diff
@@ -172,10 +172,7 @@ Each factor is independent of the other(s).
 
 **Vulnerability Scan**: A process that uses manual or automated tools to probe internal and external systems to check and report on the status of operating systems, services, and devices exposed to the network and the presence of vulnerabilities listed in the NVD, OWASP Top Ten, or SANS Top 25.
 
-**Workstation**: A device, such as a phone, tablet, or desktop or laptop computer, which is:
-
-   1. connected to the same network as CA Infrastructure and/or Network Equipment; and
-   2. capable of accessing CA Infrastructure and/or Network Equipment.
+**Workstation**: A device, such as a phone, tablet, or desktop or laptop computer, which is capable of accessing CA Infrastructure and/or Network Equipment with elevated privileges.
```

Comment:
I think this is good but upon reflection on the language, I am not sure if "elevated privileges" will always be correctly interpreted. Often, "elevated privileges" means "more than user privileges", e.g. in operating systems or Network Equipment. Here, I think we must express "elevated privileges compared to any other spot on the general internet".

This was previously reflected in the definition applying to devices "connected to the same network as CA Infrastructure and/or Network Equipment".